calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
To your question, I think I prefer labels too.  https://docs.docker.com/engine/userguide/labels-custom-metadata/ suggests that labels are an implementation of metadata, and it's labels we're going to extract, rather than generic (possibly not-yet-implemented) metadata.